words/headers.asm

```diff
@@ -1564,1073 +1564,247 @@ nt_editor_enter_screen:
-		.word xt_asm_sta_zx, z_asm_sta_zx
-		.text "sta.zx"
-
-nt_asm_sta_zxi:
-		.byte 7, IM+NN
-                .word nt_asm_stx
-		.word xt_asm_sta_zxi, z_asm_sta_zxi
-		.text "sta.zxi"
-
-nt_asm_stx:
-		.byte 3, IM+NN
-                .word nt_asm_stx_z
-		.word xt_asm_stx, z_asm_stx
-		.text "stx"
-
-nt_asm_stx_z:
-		.byte 5, IM+NN
-                .word nt_asm_stx_zy
-		.word xt_asm_stx_z, z_asm_stx_z
-		.text "stx.z"
-
-nt_asm_stx_zy:
-		.byte 6, IM+NN
-                .word nt_asm_sty
-		.word xt_asm_stx_zy, z_asm_stx_zy
-		.text "stx.zy"
-
-nt_asm_sty:
-		.byte 3, IM+NN
-                .word nt_asm_sty_z
-		.word xt_asm_sty, z_asm_sty
-		.text "sty"
-
-nt_asm_sty_z:
-		.byte 5, IM+NN
-                .word nt_asm_sty_zx
-		.word xt_asm_sty_z, z_asm_sty_z
-		.text "sty.z"
-
-nt_asm_sty_zx:
-		.byte 6, IM+NN
-                .word nt_asm_stz
-		.word xt_asm_sty_zx, z_asm_sty_zx
-		.text "sty.zx"
-
-nt_asm_stz:
-		.byte 3, IM+NN
-                .word nt_asm_stz_x
-		.word xt_asm_stz, z_asm_stz
-		.text "stz"
-
-nt_asm_stz_x:
-		.byte 5, IM+NN
-                .word nt_asm_stz_z
-		.word xt_asm_stz_x, z_asm_stz_x
-		.text "stz.x"
-
-nt_asm_stz_z:
-		.byte 5, IM+NN
-                .word nt_asm_stz_zx
-		.word xt_asm_stz_z, z_asm_stz_z
-		.text "stz.z"
-
-nt_asm_stz_zx:
-		.byte 6, IM+NN
-                .word nt_asm_tax
-		.word xt_asm_stz_zx, z_asm_stz_zx
-		.text "stz.zx"
-
-nt_asm_tax:
-		.byte 3, IM+NN
-                .word nt_asm_tay
-		.word xt_asm_tax, z_asm_tax
-		.text "tax"
-
-nt_asm_tay:
-		.byte 3, IM+NN
-                .word nt_asm_trb
-		.word xt_asm_tay, z_asm_tay
-		.text "tay"
-
-nt_asm_trb:
-		.byte 3, IM+NN
-                .word nt_asm_trb_z
-		.word xt_asm_trb, z_asm_trb
-		.text "trb"
-
-nt_asm_trb_z:
-		.byte 5, IM+NN
-                .word nt_asm_tsb
-		.word xt_asm_trb_z, z_asm_trb_z
-		.text "trb.z"
-
-nt_asm_tsb:
-		.byte 3, IM+NN
-                .word nt_asm_tsb_z
-		.word xt_asm_tsb, z_asm_tsb
-		.text "tsb"
-
-nt_asm_tsb_z:
-		.byte 5, IM+NN
-                .word nt_asm_tsx
-		.word xt_asm_tsb_z, z_asm_tsb_z
-		.text "tsb.z"
-
-nt_asm_tsx:
-		.byte 3, IM+NN
-                .word nt_asm_txa
-		.word xt_asm_tsx, z_asm_tsx
-		.text "tsx"
-
-nt_asm_txa:
-		.byte 3, IM+NN
-                .word nt_asm_txs
-		.word xt_asm_txa, z_asm_txa
-		.text "txa"
-
-nt_asm_txs:
-		.byte 3, IM+NN
-                .word nt_asm_tya
-		.word xt_asm_txs, z_asm_txs
-		.text "txs"
-
-nt_asm_tya:
-		.byte 3, IM+NN
-                .word nt_asm_arrow
-		.word xt_asm_tya, z_asm_tya
-		.text "tya"
+; "lda.#" is "xt_adm_lda_h"). All opcode words are immediate and never-native.
+;
+; Each opcode shows the traditional mnemonic as a comment with abbreviations:
+;       llhh - a 16bit little endian address
+;       zp - an 8bit zero page address
+;       #dd - an 8bit data value
+;
+; Currently the extended opcodes rmbN, smbN, bbrN, and bbsN aren't supported.
+;
+; The .nt_asm macro ensures that each word's XT points to one of a list
```

Comment:
It does introduce a macro so that it can dynamically pick the right entrypoint from the repeated jsr table 😬